halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/DefaultControllerValues.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.controllers.v1;
+
+public class DefaultControllerValues {
```

Comment:
For development purposes I have the default validate behavior set to "false", which won't be the case when this is released. This makes switching this back easier.